Calibration/HcalAlCaRecoProducers/src/AlCaIsoTracksProducer.cc

```diff
@@ -197,11 +196,18 @@ AlCaIsoTracksProducer::AlCaIsoTracksProducer(const edm::ParameterSet& iConfig) :
 
   //create also IsolatedPixelTrackCandidateCollection which contains isolation info and reference to primary track
   produces<reco::HcalIsolatedTrackCandidateCollection>("HcalIsolatedTrackCollection");
```

Comment:
@bsunanda as I just wrote, I do not see major issues. In the +1, I will point out the things to be further corrected. The only missing part are the new event sizes after the change of the output commands. Again, no big issues, just to have the full picture.

HTH